### PR TITLE
Editorial: More concise intro text (recovery)

### PR DESCRIPTION
**Pull request by mirjak**
Just a proposal but I would prefer to remove the advertising slang ("decades of transport and security experience" and "makes it attractive").

### CODE REVIEW

**Comment by MikeBishop** [comment]
Isn't the introduction the right place for ad copy, though?  It describes what the document hopes to achieve and why it's worth doing.  I think some of the rewording you've done here is an improvement, but I'm not convinced we should edit out the fact that we're trying to incorporate many years of work that hasn't been universally deployable.

File: draft-ietf-quic-recovery.md [hunk 1]
```diff
@@ -90,17 +90,13 @@ code and issues list for this draft can be found at
...
-control and loss recovery, and where applicable, attributes the TCP equivalent
-in RFCs, Internet-drafts, academic papers, and/or TCP implementations.
-
+QUIC is a new multiplexed and secure transport protocol atop UDP, specified in
+{{QUIC-TRANSPORT}}. This document describes congestion control and loss
+recovery for QUIC. Mechanisms described in this documnet follow the spirit
+of existing TCP congestion control and loss recovery mechanisms, described in
+RFCs, various Internet-drafts, or academic papers, and also those prevalent in
+the Linux TCP implementation. Where applicable, this document attributes the TCP
+equivalent.
```
Comment (martinthomson) [inline draft-ietf-quic-recovery.md]: This last sentence seems incomplete.  I think that you are saying that the mechanisms here are informed by TCP.  Any differences are necessary as a result of differences in QUIC relative to TCP; or demonstrably advantageous as a result of QUIC lacking some of the limitations of TCP.

File: draft-ietf-quic-recovery.md [hunk 2]
```diff
@@ -90,17 +90,13 @@ code and issues list for this draft can be found at
...
-
-QUIC implements the spirit of existing TCP congestion control and loss recovery
-mechanisms, described in RFCs, various Internet-drafts, and also those prevalent
-in the Linux TCP implementation.  This document describes QUIC congestion
-control and loss recovery, and where applicable, attributes the TCP equivalent
-in RFCs, Internet-drafts, academic papers, and/or TCP implementations.
-
+QUIC is a new multiplexed and secure transport protocol atop UDP, specified in
+{{QUIC-TRANSPORT}}. This document describes congestion control and loss
+recovery for QUIC. Mechanisms described in this documnet follow the spirit
```
Comment (janaiyengar) [inline draft-ietf-quic-recovery.md]:
```suggestion
recovery for QUIC. Mechanisms described in this document follow the spirit
```

File: draft-ietf-quic-recovery.md [hunk 3]
```diff
@@ -90,17 +90,13 @@ code and issues list for this draft can be found at
...
+of existing TCP congestion control and loss recovery mechanisms, described in
+RFCs, various Internet-drafts, or academic papers, and also those prevalent in
+the Linux TCP implementation. Where applicable, this document attributes the TCP
```
Comment (janaiyengar) [inline draft-ietf-quic-recovery.md]:
```suggestion
the Linux TCP implementation.
```

File: draft-ietf-quic-recovery.md (hunk 1 above)
Comment (janaiyengar) [inline draft-ietf-quic-recovery.md]:
```suggestion
```

Comment (janaiyengar) [reply draft-ietf-quic-recovery.md]: I don't think this last sentence is useful or necessary.

Comment (mirjak) [reply draft-ietf-quic-recovery.md]: I just took that from the old intro. Removed now.

File: draft-ietf-quic-recovery.md [hunk 4]
```diff
@@ -90,17 +90,12 @@ code and issues list for this draft can be found at
...
-in the Linux TCP implementation.  This document describes QUIC congestion
-control and loss recovery, and where applicable, attributes the TCP equivalent
-in RFCs, Internet-drafts, academic papers, and/or TCP implementations.
-
+QUIC is a new multiplexed and secure transport protocol atop UDP, specified in
+{{QUIC-TRANSPORT}}. This document describes congestion control and loss
+recovery for QUIC. Mechanisms described in this document follow the spirit
+of existing TCP congestion control and loss recovery mechanisms, described in
+RFCs, various Internet-drafts, or academic papers, and also those prevalent in
+the Linux TCP implementation.
```
Comment (martinthomson) [inline draft-ietf-quic-recovery.md]: Do we need to privilege Linux, or can we simply say "TCP implementations"?

```suggestion
TCP implementations.
```

**Pull request merged**